microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Securely debug original code by using Azure Artifacts symbol server source maps
+description: Learn how to configure DevTools to use source maps from the Azure Artifacts symbol server to securely debug your original source code in DevTools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 03/02/2022
+---
+
+# Securely debug original code by using Azure Artifacts symbol server source maps
+
+To securely see and work with your original development source code in DevTools rather than the compiled, minified, and bundled production code that's returned by the web server, you can use source maps published on the Azure Artifacts symbol server.
+
+Source maps map your compiled production code to your original development source files. In DevTools, you can then see and work with your familiar development source files, instead of the compiled code. To learn more about source mapping and using source maps in DevTools, see [Map the processed code to your original source code, for debugging](source-maps.md).
+
+Downloading your source maps from the Azure Artifacts symbol server makes it possible to debug your production website by retrieving your original development source code securely.
+
+
+<!-- ====================================================================== -->
+## Prerequisite: Publish source maps to the Azure Artifacts symbol server
+
+To use source maps from the Azure Artifacts symbol server in DevTools, they need to be published to the server first.
+
+To learn about how to publish source maps, see [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](publish-source-maps-to-azure.md).
+
+
+<!-- ====================================================================== -->
+## Step 1: Generate a Personal Access Token for Azure DevOps
+
+To obtain source maps from the Azure Artifacts symbol server, DevTools uses the fetch protocol to talk to Azure DevOps, which requires a valid Personal Access Token (PAT).
+
+To generate a PAT in Azure DevOps:
+
+1. Sign in to your Azure DevOps organization by going to `https://dev.azure.com/{yourorganization}`.
+
+1. In Azure DevOps, go to **User settings** > **Personal access tokens**:
+    
+   ![The 'User settings' menu in Azure DevOps, with the 'Personal access tokens' command.](images/ado-pat-settings.png)
+
+   The **Personal Access Tokens** page appears:
+
+   ![The 'Personal Access Tokens' page in Azure DevOps.](images/ado-pat-page.png)
+
+1. Click **New Token**.  The **Create a new personal access token** dialog box opens:
+
+   ![The 'Create a new personal access token' dialog box, with 'Read' scope for Symbols selected.](images/ado-pat-config-read.png)
+
+1. In the **Name** text box, enter a name for the PAT, such as **DevTools source maps**.
```

Comment:
```suggestion
1. In the **Name** text box, enter the name for the PAT that you previously created per [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](publish-source-maps-to-azure.md), such as **DevTools source maps**.
```

Reply:
No the user should invent a name that they can later remember. And they shouldn't use the same name as on the other page. These are 2 different PATs. One will likely be created by the CI admin, while the other will be created by the user of DevTools.

Reply:
Both pages (of this Merged PR) use identical name in screenshot (devtools source maps) & text (DevTools source maps).  Should add text maybe on both pages, like:

"Invent a name that you can later remember.  Don't|You shouldn't use the same name as on the Publish|Consume step/article. The Publish PAT & the Consume PAT are 2 different PATs.   The Publish PAT is typically created by the Continuous Integration (CI) admin, while the Consume PAT will be created by the user of DevTools."

Reply:
Addressing this in #1783.